etl/steps/data/garden/wb/2023-07-10/education.py

```diff
@@ -100,22 +100,34 @@ def run(dest_dir: str) -> None:
     ds_garden.save()
 
 
-def combine_historical_literacy_expenditure(tb):
+def combine_historical_literacy_expenditure(tb, tb_literacy, tb_expenditure):
     """
-    Combines historical and recent literacy and expenditure data into a single DataFrame.
+    Merges historical and recent literacy and expenditure data into a single DataFrame.
```

Comment:
Style: In Python, the first line in a docstring should be imperative, e.g. "Merge historical...".

Reply:
DataFrame -> Table?